engine/graphics/src/vulkan/graphics_vulkan.cpp

```diff
@@ -148,6 +148,15 @@ namespace dmGraphics
         m_TextureFormatSupport   |= 1 << TEXTURE_FORMAT_RGB;
         m_TextureFormatSupport   |= 1 << TEXTURE_FORMAT_RGBA;
 
+        m_TextureFormatSupport   |= 1 << TEXTURE_FORMAT_RGB16F;
+        m_TextureFormatSupport   |= 1 << TEXTURE_FORMAT_RGB32F;
+        m_TextureFormatSupport   |= 1 << TEXTURE_FORMAT_RGBA16F;
+        m_TextureFormatSupport   |= 1 << TEXTURE_FORMAT_RGBA32F;
+        m_TextureFormatSupport   |= 1 << TEXTURE_FORMAT_R16F;
+        m_TextureFormatSupport   |= 1 << TEXTURE_FORMAT_RG16F;
+        m_TextureFormatSupport   |= 1 << TEXTURE_FORMAT_R32F;
+        m_TextureFormatSupport   |= 1 << TEXTURE_FORMAT_RG32F;
+
```

Comment:
I'm more uncertain of these, but it seems to be able to create e.g. a `TEXTURE_FORMAT_RG32F` render target.